pkg/server/server.go

```diff
@@ -36,9 +36,9 @@ import (
 )
 
 const (
-	invalidTrustDomainAttestedNode = "An attested node with trust domain '%v' has been detected, " +
+	invalidTrustDomainAttestedNode = "an attested node with trust domain '%v' has been detected, " +
 		"which does not match the configured trust domain of '%v'. Agents may need to be reconfigured to use new trust domain"
-	invalidTrustDomainRegistrationEntry = "A registration entry with trust domain '%v' has been detected, " +
+	invalidTrustDomainRegistrationEntry = "a registration entry with trust domain '%v' has been detected, " +
```

Comment:
This is also used to log, of which our convention is leading uppercase...

Reply:
Unlike the previous one, this one is not used to log directly, but in fmt.Errorf which go staticcheck complains about

Reply:
This change is fine. I'm doing a small cleanup refactor of server.go to ease some upcoming testing and I'll see if I can't figure out something consistent here. One idea is that the logging can be rich but the returned error doesn't have to be since it is just logged before exit by the CLI layer.